app/src/components/TermsOfUse.jsx

```diff
@@ -2,355 +2,359 @@ import React, { Component } from 'react';
+                  DAMAGES, LOSSES, AND CAUSES OF ACTION ARISING OUT OF OR
+                  RELATING TO THESE TERMS OR YOUR USE OF THE SITE (WHETHER
+                  IN CONTRACT, TORT (INCLUDING NEGLIGENCE), WARRANTY OR
+                  OTHERWISE) EXCEED ONE HUNDRED DOLLARS ($100 USD).
+                </li>
+                <li>
+                  Savings Clause. CERTAIN JURISDICTIONS DO NOT ALLOW EXCLUSIONS OR LIMITATIONS
+                  ON CERTAIN DAMAGES AND LIABILITIES. IF YOU RESIDE IN SUCH A JURISDICTION,
+                  SOME OR ALL OF THE ABOVE EXCLUSIONS OR LIMITATIONS MAY NOT APPLY TO YOU,
+                  AND YOU MAY HAVE ADDITIONAL RIGHTS. THE LIMITATIONS OR EXCLUSIONS LIABILITY
+                  CONTAINED IN THESE TERMS APPLY TO YOU TO THE FULLEST EXTENT SUCH LIMITATIONS
+                  OR EXCLUSIONS ARE PERMITTED UNDER THE LAWS OF THE JURISDICTION IN WHICH YOU ARE LOCATED.
+                </li>
+              </ol>
+            </li>
+            <li>
+              <h2>Digital Millennium Copyright Act Compliance</h2>
+              <p>
+                If You are a copyright owner or an agent thereof, and believe that any Site
+                content infringes upon Your copyrights, You may submit a notification
+                pursuant to the Digital Millennium Copyright Act (“DMCA”) by providing
+                our Copyright Agent (as defined below) with the following information
+                in writing (see 17 U.S.C § 512(c)(3) for further detail):
+              </p>
+              <ol>
+                <li>
+                  A physical or electronic signature of a person authorized
+                  to act on behalf of the owner of an exclusive right that is allegedly infringed;
+                </li>
+                <li>
+                  Identification of the exact location on the Site of copyrighted work
+                  claimed to have been infringed, or, if multiple copyrighted works on the
+                  Site are covered by a single notification, a representative list of such works as located on the Site;
+                </li>
+                <li>
+                  The address, telephone number and electronic mail address of the complaining party
+                </li>
+                <li>
+                  A statement that the complaining party has
+                  a good faith belief that use of the material in the
+                  manner complained of is not authorized by the copyright owner, its agent or the law
+                </li>
+                <li>Your electronic or physical signature (as appropriate).</li>
+              </ol>
+              <p>
+                The designated Copyright Agent to receive notifications of claimed infringement is:
+                SkyTruth Tel: <a href="tel:1 (304) 885-4581">+1 (304) 885-4581</a>
+                Email: <a href="dmcaagent@skytruth.org">dmcaagent@skytruth.org</a>.
```

Comment:
You should probably avoid to add an email in plain because of spam bots but I don't know how clever they are.
Also, should we add `mailto:` at the beginning of the address?